Dockerfile-default-rocm

```diff
@@ -1,40 +1,138 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
+#RUN sudo sysctl -w kernel.core_pattern="core-%h-%e-%s-%t-%u-%p"
+
+#RUN dpkg -S /opt/ompi
+RUN apt remove -y openmpi ucx
+#RUN mv /opt/ompi /opt/ompi.0 && mv /opt/ucx /opt/ucx.0
+RUN mv /opt/ompi /opt/ompi.0 
+#RUN ln -s /container/ompi /opt && ln -s /container/ucx /opt
+RUN ln -s /container/ompi /opt 
+#RUN ls /opt/ompi && exit 99
 
 RUN mkdir -p /var/run/sshd
+#RUN pip install torch torchvision torchaudio
 RUN rm /etc/apt/sources.list.d/rocm.list
 RUN apt-get update \
 	&& DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-		autoconf \
-		automake \
-		autotools-dev \
-		build-essential \
-		ca-certificates \
-		curl \
-		daemontools \
-		libkrb5-dev \
-		libssl-dev \
-		libtool \
+                autoconf \
+                automake \
+                autotools-dev \
+                build-essential \
+                ca-certificates \
+                curl \
+                daemontools \
+                glibc-source \
+                ibverbs-providers \
+                libibverbs1 \
+                libkrb5-dev \
+                librdmacm1 \
+                libssl-dev \
+                libtool \
 		git \
 		krb5-user \
 		cmake \
-		g++-4.8 \
+		g++ \
```

Comment:
should this be locked to a specific version of g++?

Reply:
A newer version was needed but we're open to a pinned version.